boards/stm32f4discovery/Makefile.include

```diff
@@ -5,6 +5,9 @@ INCLUDES += -I$(RIOTBOARD)/common/stm32/include
 PORT_LINUX ?= /dev/ttyUSB0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 
+# setup emulator
+include $(RIOTMAKE)/tools/renode.inc.mk
+
```

Comment:
Not needed, the emulator is included in Makefile.include:

https://github.com/RIOT-OS/RIOT/blob/8aae7f165f24b23d0afffb560f761cf42af6c329/Makefile.include#L400-L405

```suggestion
```